### DISCUSSION TITLE
Extractors besides Archive.org not working on Windows install

### DISCUSSION THREAD

**Discussion by FoxWMulder** [ArchiveBox/ArchiveBox#1114] [General]
Installed on Windows. Launched interface at http://127.0.0.1:8000/
When saving a site to an archive, it saves only the archive.org version. In the snapshot section, it is not available to see that it does not save in any other format. How can I make it save the site to my computer?
In the configuration file, for example, saving to WARC is enabled.

**Comment by pirate** [comment]
What is the CLI output during archiving? Does it show errors from the other extractors failing?

FYI windows is not officially supported unless you are running it in Docker.

**Reply by FoxWMulder** [reply]
Unfortunately. I was never able to start ArchiveBox. Neither under Docker nor under Ubuntu. Complicated thing. Thank for your answer.

**Reply by pirate** [reply]
I'd advise using a different tool other than ArchiveBox then, check out some alternatives here: https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#other-archivebox-alternatives

**Reply by TomLucidor** [reply]
Probably asking for a Chocolatey installer would make things easier?

**Reply by FoxWMulder** [reply]
Don't know. but whoever writes the installation instructions needs to be hit on the head. I installed Docker on Windows. Opened http://127.0.0.1:8000. I see the interface. How to create a user is COMPLETELY unclear. Where to enter the command "archivebox manage changepassword USERNAME"? When I try to enter the command to create a user (if I enter it where necessary), for some reason Docker writes “Arhivebox should never be run as root”. What does root have to do with it???

Okay, I created a user. How to set a password? The most interesting thing is that I cannot find an answer to this seemingly simple question on the Internet.

A tool for masochists.

**Reply by pirate** [reply]
The exact commands you need to run are all in the README:

https://github.com/ArchiveBox/ArchiveBox#%EF%B8%8F-easy-setup


Or use something else instead of archivebox

> I'd advise using a different tool other than ArchiveBox then, check out some alternatives here: https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#other-archivebox-alternatives

Or perhaps contribute something to improve the instructions instead of attacking the people who make your software for free?

**Reply by TomLucidor** [reply]
@pirate I ain't attacking, just having ideas since the pip install does not cover Windows, which weirds me out a bit. Considering to play around with Docker later tho

**Reply by FoxWMulder** [reply]
Of course I looked at the teams here https://github.com/ArchiveBox/ArchiveBox#%EF%B8%8F-easy-setup

1) Where should I enter them? 2) Let's say I found where, but when I try to enter the command "archivebox manage createsuperuser" the error "ArchiveBox should never be run as root!" appears. Perhaps the command is not suitable for the current version of ArchiveBox?

**Reply by pirate** [reply]
Pip install does not cover windows because windows is a massively different OS than Linux and macOS, and I don't use windows so I can't provide support for it.

`docker compose run archivebox init --setup`

Is all you need to run in Docker on any OS, it will walk you through creating a superuser.

If you need any other ArchiveBox commands run them using `run` not `exec`, e.g.:
  
`docker compose run archivebox help`  
`docker compose run archivebox add ...`
